frontend/e-learning-fe/src/components/answer.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import Choices from "./choices";
+import API from "../api";
+
+function Answer() {
+  const [words, setWords] = useState([]);
+  const [category, setCategory] = useState([]);
+  const [flag, setFlag] = useState(1);
+
+  const token = localStorage.getItem("token");
+  const category_id = 1;
+
+  const fetchData = async () => {
+    await API.word
+      .getWordsPerCategory({
+        id: category_id,
+        token,
+      })
+      .then((data) => {
+        setFlag(1 + 1);
```

Comment:
what is this for?

Reply:
To control the useEffect sir

Reply:
if im reading this correctly the flow is like this
user visited page
`fetchData` is called
`flag` is updated to `2`
`fetchData` is called again << why do we need to call it again?